I'm sorry, but an uncaught exception occurred.

While parsing E:\gal\Ha&Blade/game/deadending.rpy.
UnicodeDecodeError: 'utf8' codec can't decode byte 0xd5 in position 31: invalid continuation byte

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "E:\renpy-6.99.10-sdk\renpy\bootstrap.py", line 277, in bootstrap
    renpy.main.main()
  File "E:\renpy-6.99.10-sdk\renpy\main.py", line 354, in main
    renpy.game.script.load_script() # sets renpy.game.script.
  File "E:\renpy-6.99.10-sdk\renpy\script.py", line 259, in load_script
    self.load_appropriate_file(".rpyc", ".rpy", dir, fn, initcode)
  File "E:\renpy-6.99.10-sdk\renpy\script.py", line 723, in load_appropriate_file
    data, stmts = self.load_file(dir, fn + source)
  File "E:\renpy-6.99.10-sdk\renpy\script.py", line 538, in load_file
    stmts = renpy.parser.parse(fullfn)
  File "E:\renpy-6.99.10-sdk\renpy\parser.py", line 2417, in parse
    lines = list_logical_lines(fn, filedata, linenumber)
  File "E:\renpy-6.99.10-sdk\renpy\parser.py", line 185, in list_logical_lines
    data = f.read()
  File "/home/tom/ab/x64lucid-deps/install/lib/python2.7/codecs.py", line 674, in read
  File "/home/tom/ab/x64lucid-deps/install/lib/python2.7/codecs.py", line 480, in read
UnicodeDecodeError: 'utf8' codec can't decode byte 0xd5 in position 31: invalid continuation byte

Windows-XP-5.1.2600-SP3
Ren'Py 6.99.10.1227
 
